apps/hubble/src/hubble.ts

```diff
@@ -1453,17 +1452,11 @@ export class Hub implements HubInterface {
     // Check if we already have this client
     const result = this.syncEngine.addContactInfoForPeerId(peerId, message);
     if (result.isOk() && !this.performedFirstSync) {
-      // Should only sync last ~day worth of messages with new peers. For now, only sync with the first peer so we are upto
-      // date on startup.
-      log.debug({ peerInfo: message }, "New peer but only performing first sync");
-      const syncResult = await ResultAsync.fromPromise(
-        this.syncEngine.diffSyncIfRequired(this, peerId.toString()),
-        (e) => e,
-      );
-      if (syncResult.isErr()) {
-        log.error({ error: syncResult.error, peerId }, "Failed to sync with new peer");
-      }
+      // Sync with the first peer so we are upto date on startup.
       this.performedFirstSync = true;
+      setTimeout(async () => {
+        await ResultAsync.fromPromise(this.syncEngine.diffSyncIfRequired(this), (e) => e);
+      }, 10 * 1000);
```

Comment:
nit: can we make this a constant with unit suffix? e.g. 
```suggestion
      const DIFF_SYNC_TIMEOUT_MS = 10 * 1000;
```